crates/forge/tests/data/targets/only_integration/Scarb.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "targets"
```

Comment:
Won't you get warnings from Scarb when using snforge_std as git dep if you use the same name for all this packages?